### DISCUSSION TITLE
Basic Q

### DISCUSSION THREAD

**Discussion by MAAB-FW** [Alex313031/thorium#648] [Q&A]
Is it possible to implement the extra features of edge browser to thorium?

**Comment by midzer** [comment]
"Possible" is everything.

What are those extra features of Edge browser you are talking of?

**Reply by MAAB-FW** [reply]
1. The built-in optimized screenshot feature which supports long screenshots(manually or automatically), (very useful)
2. And there's an option to "show bookmarks only" on the bookmark bar. (very useful)
3. Full control over downloaded files in the downloads tab, (very useful)
4. A sidebar for easy navigation, (useful)
5. And a "magnify image" option on the right click options —also can activated by double-clicking the "Ctrl" key. (useful)
6. Browser essentials - for save power with efficiency mode. (...)
7. Launch Search bar - win+shift+f  (rarely use)
8. (It's firefox feature) when installing any extension a pop will show for using those extension in private(incognito) tab.
etc. 
Those are greatly increase productivity. Me and some of my friends can't exit edge browser because of such feature.
That's for today.
